### DISCUSSION TITLE
WatchOS support

### DISCUSSION THREAD

**Discussion by SteveCutler** [buhe/langchain-swift#87] [General]
Will this run on watchOS?

**Comment by buhe** [comment]
@SteveCutler I'll give it a try. Can you raise an issue?

**Comment by buhe** [comment]
I am try at https://github.com/buhe/langchain-swift/pull/89,

good news is compiled, and bad is requires conditional compilation.

**Reply by SteveCutler** [reply]
That's great, what did you have to change to get it to run?

**Reply by buhe** [reply]
Use 0.45.0+ please.
